dp/generate_climos.py

```diff
@@ -39,6 +39,86 @@
 cdo = Cdo()
 
 
+def has_climo_periods(input_file, climo):
+    '''
+    This function checks if the given input NetCDF dataset has a variable "climo_periods".
+    generate_climos requires the input dataset to have the variable, thus this function will
+    warn if there is no such variable.
+    '''
+    periods = input_file.climo_periods.keys() & climo
+    logger.info("climo_periods: {}".format(periods))
+    if len(periods) == 0:
+        logger.info(f"{input_file.filepath()} has no variable 'climo_periods'")
+    return periods
+
+
+def is_NetCDF(filepath):
```

Comment:
Two comments here:
1. Again, the name here is suggesting we would get some sort of `boolean` out of it.  `get_cdfdataset` may work better here? I'll leave it up to you.
2. We don't want to have capital letters in our method names unless we are using camelCaps.

Reply:
Agreed with point 1.

Reply:
Echoes my thoughts.